rtrlib/rtr_mgr.h

```diff
@@ -73,7 +73,7 @@ typedef void (*rtr_mgr_status_fp)(const struct rtr_mgr_group *, enum rtr_mgr_sta
 /**
  * Type alias \c pfx_for_each_fp for function signature: \code (*)(const struct pfx_record *pfx_record, void *data) \endcode
  */
-typedef void (*pfx_for_each_fp)(const struct pfx_record *pfx_record, void *data);
+//typedef void (*pfx_for_each_fp)(const struct pfx_record *pfx_record, void *data);
```

Comment:
If the typedef is really not used it should be removed. I think there is no reason to keep commented out code in times of revision control systems :-)